### PR TITLE
To Do List Structure files

### PR DESCRIPTION
**Pull request by Chimoneg27**
### For this feature I did the following

- Setup a webapack environment. Quick note, I had to build a new one, the one I submitted for the previous exercise had some issues
- Made sure to remove linter errors
- Used webpack to organize my code
- Added to do list structure
- Used basic simple styling in CSS
- Made sure project runs properly when typing in "npm run build" in the command line

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,123 @@
...
+- **[An input field]**
+- **[Dynamically created content]**
+- **[Webpack configuration]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- **[Coming soon]**
+
```
Comment (HAMMAS-SALEEM) [inline README.md]: - [ ] It would be nice if you remove the Live Demo section since you don't have any working live link 🤔

**Review by HAMMAS-SALEEM — COMMENTED** [review]
# Invalid Submission :red_circle:

invalid-submission

## Hey @Chimoneg27 :wave:

Good job on implementing some of the requirements but I cannot proceed review of your pull request because there are some issues that you still need to work on to go to the next project but you are almost there! :blush:

# Optional

_All the changes with optional tags are not crucial enough to prevent you from getting the approval. Still, we highly recommend to students that they should take these changes into consideration._

Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

---------------------------------------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by ahmetbozaci** [comment]
### Hello @Chimoneg27 
### _This is an additional review as a result of the second opinion request._ 

- Your readme file is already professional. While there is no live demo link, it is ok to write 'coming soon' text. End of the project you can update the link.

- Kindly submit it for review on your dashboard. The next code reviewer will check it.





___


### Cheers and Happy coding! 💻 👏
____
[GitHub](https://github.com/ahmetbozaci) | [LinkedIn](https://linkedin.com/in/ahmetbozaci) | [Twitter](https://twitter.com/ahmtbozaci)

_Reviewed by [Ahmet Bozacı ](https://github.com/ahmetbozaci)_

**Review by rica213 — APPROVED** [review]
# 🟢 STATUS: APPROVED
Hi Garvin, 
My name is Larissa. I am your code reviewer 🥰

Your project is complete🛸 🛸 
- Webpack well configured ✔️ 
- Dynamically generate a list of to-dos ✔️ 
- Minimal CSS on the page ✔️ 

Congratulations! 🎉  And keep up the great work ❤️ 




## Highlights

- [ ] No Linters errors ✔️ 
- [ ] Descriptive PR ✔️ 
- [ ] Professional Documentation ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**